README.md

```diff
@@ -165,3 +166,38 @@ tkn pipeline start operator-hosted-pipeline \
   --workspace name=registry-credentials,secret=my-registry-secret \
   --showlog
 ```
+
+
+## Operator Release pipeline
+The Release pipeline runs after the layers of validation (CI (optionally) and Hosted pipeline).
+It is used to certify and publish submitted bundle version.
+It is triggered by merging the submission pull request and sucessfully completes with updating the GitHub
+indicating published versions.
```

Comment:
How about this instead?

> It is triggered by a merged pull request and successfully completes once the bundle has been distributed to all relevant catalogs and appears in the Red Hat Ecosystem Catalog.

It's probably more important to state that it succeeds when the distribution is complete. The github statuses are just a nice-to-have feature.